Bump SixLabors.ImageSharp due to vulnerabilities

```
warning NU1902: Package 'SixLabors.ImageSharp' 3.1.3 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-5x7m-6737-26cr
```

--- a/Directory.Build.targets
+++ b/Directory.Build.targets
@@ -23,18 +23,18 @@
     <PackageReference Update="Nito.AsyncEx.Coordination" Version="5.1.2" />
 
     <PackageReference Update="AspNet.Security.OAuth.Discord" Version="8.0.0" />
-    
+
     <PackageReference Update="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="8.0.3" />
     <PackageReference Update="Microsoft.AspNetCore.SpaServices.Extensions" Version="8.0.3" />
-    <PackageReference Update="Microsoft.Extensions.Caching.Memory" Version="8.0.0" />
-    <PackageReference Update="Microsoft.Extensions.Configuration.UserSecrets" Version="8.0.0" />
-    <PackageReference Update="Microsoft.Extensions.DependencyInjection" Version="8.0.0" />
-    <PackageReference Update="Microsoft.Extensions.Hosting" Version="8.0.0" />
-    <PackageReference Update="Microsoft.Extensions.Hosting.Abstractions" Version="8.0.0" />
-    <PackageReference Update="Microsoft.Extensions.Http" Version="8.0.0" />
+    <PackageReference Update="Microsoft.Extensions.Caching.Memory" Version="8.0.1" />
+    <PackageReference Update="Microsoft.Extensions.Configuration.UserSecrets" Version="8.0.1" />
+    <PackageReference Update="Microsoft.Extensions.DependencyInjection" Version="8.0.1" />
+    <PackageReference Update="Microsoft.Extensions.Hosting" Version="8.0.1" />
+    <PackageReference Update="Microsoft.Extensions.Hosting.Abstractions" Version="8.0.1" />
+    <PackageReference Update="Microsoft.Extensions.Http" Version="8.0.1" />
     <PackageReference Update="Microsoft.Extensions.Http.Polly" Version="8.0.3" />
-    <PackageReference Update="Microsoft.Extensions.Logging.Abstractions" Version="8.0.1" />
-    <PackageReference Update="Microsoft.Extensions.Logging.Console" Version="8.0.0" />
+    <PackageReference Update="Microsoft.Extensions.Logging.Abstractions" Version="8.0.2" />
+    <PackageReference Update="Microsoft.Extensions.Logging.Console" Version="8.0.1" />
     <PackageReference Update="Microsoft.Extensions.Options" Version="8.0.2" />
     <PackageReference Update="Microsoft.Extensions.Options.ConfigurationExtensions" Version="8.0.0" />
     <PackageReference Update="Microsoft.NET.Test.Sdk" Version="17.*" />
@@ -61,7 +61,7 @@
     <PackageReference Update="NUnit3TestAdapter" Version="4.5.0" />
     <PackageReference Update="Shouldly" Version="4.2.1" />
 
-    <PackageReference Update="SixLabors.ImageSharp" Version="3.1.3" />
+    <PackageReference Update="SixLabors.ImageSharp" Version="3.1.5" />
 
     <PackageReference Update="Microsoft.CodeAnalysis.CSharp" Version="4.9.2" />
     <PackageReference Update="Microsoft.CodeAnalysis.CSharp.Analyzer.Testing.NUnit" Version="1.*" />

